internal/blobtesting/map.go

```diff
@@ -25,7 +25,22 @@ type mapStorage struct {
 	keyTime map[blob.ID]time.Time
 	// +checklocks:mutex
 	timeNow func() time.Time
-	mutex   sync.RWMutex
+	// +checklocks:mutex
+	totalBytes int64
+	// +checklocks:mutex
+	limit int64
```

Comment:
`limit` doesn't change after the construction of the map, so it doesn't need to be under the lock

Reply:
switched to ignore